### ISSUE TITLE
[request] getting version list

### ISSUE THREAD

**Issue by BASgamer** [Bob123a1/CDNSP-GUI#69]
Is there anyway to get the version list without a valid cert? i would still like to use this if for that if i could do that. so that i can find when one of my games has been updated so i can go find it.

**Comment by Bob123a1** [comment]
Yup that's possible, will release a new build tomorrow with some other changes

**Comment by BASgamer** [comment]
thanks a lot love your woke
